packages/components/src/components/CodeEditor.tsx

```diff
@@ -243,6 +261,38 @@ export const CodeEditor = forwardRef<CodeEditorHandle, CodeEditorProps>(
       });
     }, [format]);
 
+    useEffect(() => {
+      editorView.current?.dispatch({
+        effects: compViewNodeListener.reconfigure(
+          keymap.of([
+            {
+              key: "Alt-Shift-v",
+              run: () => {
+                if (!onViewValuePath) {
+                  return true;
+                }
+                const offset = editorView.current?.state.selection.main.to;
+                if (offset === undefined) {
+                  return true;
+                }
+                if (sourceId === undefined) {
+                  return true;
+                }
```

Comment:
tiny - but I'd probably have done 
```
if (!offetId || !sourceId) {
```

Reply:
`sourceId` probably won't be an empty string (though it's technically allowed), but `offset` could be `0`.

Reply:
Could also do (_.isUndefined(offsetID) || _.isUndefined(sourceID).

(Though that adds the (small) dependency.